### PR TITLE
Flux 0.106

### PR DESCRIPTION
**Pull request by sanderson**
- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/v2.0/reference/flux/stdlib/experimental/integral.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+```
+
+## Parameters
+
+### unit
+Time duration used when computing the integral.
+
+_**Data type:** Duration_
+
+### interpolate
```
Comment (kelseiv) [inline content/influxdb/v2.0/reference/flux/stdlib/experimental/integral.md:36]: Is interpolate or interpolation the parameter?

File: content/influxdb/v2.0/reference/flux/stdlib/experimental/integral.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+Type of interpolation to use.
+Defaults to `""`.
+
+Use one of the following interpolation options:
+
+- _empty sting for no interpolation_
```
Comment (kelseiv) [inline content/influxdb/v2.0/reference/flux/stdlib/experimental/integral.md:42]:
```suggestion
- _empty string for no interpolation_
```

File: content/influxdb/v2.0/reference/flux/stdlib/experimental/integral.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+integral(
+  unit: 10s,
+  interpolation: ""
+)
+```
+
+## Parameters
+
+### unit
+Time duration used when computing the integral.
```
Comment (kelseiv) [inline content/influxdb/v2.0/reference/flux/stdlib/experimental/integral.md:32]:
```suggestion
Time duration used to compute the integral.
```

Comment (kelseiv) [reply content/influxdb/v2.0/reference/flux/stdlib/experimental/integral.md:32]: Would this edit change the meaning?

File: content/influxdb/v2.0/reference/flux/stdlib/experimental/integral.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+```js
+integral(
+  unit: 10s,
+  interpolation: ""
+)
+```
+
+## Parameters
+
+### unit
```
Comment (kelseiv) [inline content/influxdb/v2.0/reference/flux/stdlib/experimental/integral.md:31]: Is unit a required parameter?

**Review by kelseiv — APPROVED** [review]

File: content/influxdb/v2.0/reference/flux/stdlib/experimental/integral.md (hunk 4 above)
Comment (sanderson) [reply content/influxdb/v2.0/reference/flux/stdlib/experimental/integral.md:31]: It is. Added the required flag.

File: content/influxdb/v2.0/reference/flux/stdlib/experimental/integral.md (hunk 3 above)
Comment (sanderson) [reply content/influxdb/v2.0/reference/flux/stdlib/experimental/integral.md:32]: Nope, it's a good change. Thanks!

File: content/influxdb/v2.0/reference/flux/stdlib/experimental/integral.md (hunk 1 above)
Comment (sanderson) [reply content/influxdb/v2.0/reference/flux/stdlib/experimental/integral.md:36]: It's `interpolate`. I updated the code block above. Nice catch!

**Pull request merged**